aspnetcore/host-and-deploy/visual-studio-publish-profiles.md

```diff
@@ -14,64 +14,40 @@ By [Sayed Ibrahim Hashimi](https://github.com/sayedihashimi) and [Rick Anderson]
 
 This document focuses on using Visual Studio 2017 or later to create and use publish profiles. The publish profiles created with Visual Studio can be run from MSBuild and Visual Studio. See [Publish an ASP.NET Core web app to Azure App Service using Visual Studio](xref:tutorials/publish-to-azure-webapp-using-vs) for instructions on publishing to Azure.
 
-The following project file was created with the command `dotnet new mvc`:
-
-::: moniker range=">= aspnetcore-2.2"
-
-```xml
-<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-  </ItemGroup>
-
-</Project>
-```
-
-::: moniker-end
-
-::: moniker range="< aspnetcore-2.2"
+The `dotnet new mvc` command produces a project file containing the following top-level `<Project>` element:
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-  </ItemGroup>
-
+    <!-- omitted for brevity -->
 </Project>
 ```
 
-::: moniker-end
-
-The `<Project>` element's `Sdk` attribute accomplishes the following tasks:
+The preceding `<Project>` element's `Sdk` attribute accomplishes the following tasks:
 
-* Imports the properties file from *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.Props* at the beginning.
-* Imports the targets file from *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.targets* at the end.
+* Imports the [MSBuild properties](/visualstudio/msbuild/msbuild-properties) file from *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.props* at the beginning.
+* Imports the [MSBuild targets](/visualstudio/msbuild/msbuild-targets) file from *$(MSBuildSDKsPath)\Microsoft.NET.Sdk.Web\Sdk\Sdk.targets* at the end.
 
-The default location for `MSBuildSDKsPath` (with Visual Studio 2017 Enterprise) is the *%programfiles(x86)%\Microsoft Visual Studio\2017\Enterprise\MSBuild\Sdks* folder.
+The default location for `MSBuildSDKsPath` (with Visual Studio 2019 Enterprise) is the *%programfiles(x86)%\Microsoft Visual Studio\2019\Enterprise\MSBuild\Sdks* folder.
 
-The `Microsoft.NET.Sdk.Web` SDK depends on:
+The `Microsoft.NET.Sdk.Web` SDK (Web SDK) depends on the following SDKs:
```

Comment:
I think this is correct now, but it feels inevitable that it will become wrong. Do folks need/want this level of detail here? Or is this something that we could abstract away? It looks like only the .NET Core SDK and Razor SDK gets their own special documentation, so maybe the others shouldn't be listed?